test_regress/t/t_sampled_expr_bad.pl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2019 by Wilson Snyder. This program is free software; you
```

Comment:
Please fix the copyright, rename this test to `t_sampled_expr_unsup.pl`, and add an output comparison. See other tests, look for `expect_filename => $Self->{golden_filename}`. You need a `t_sampled_expr_unsup.out` file for it, but don't edit it manually, just `touch` it and then run the test with `--golden`.

Reply:
Thanks for detailed instruction. Done.